### ISSUE TITLE
Error running Pylint

### ISSUE THREAD

**Issue by andb0t** [AtomLinter/linter-pylint#309]
When a pylint run is triggered, pylint seems to work (it shows and underlines linting problems), but the familiar error msg is popping up with this log in the console:
```
[Linter] Error running Pylint Error: '"-"' is not recognized as an internal or external command,
operable program or batch file.         linter-registry.js:145
    at Object.<anonymous> (main.js:161)
    at Generator.next (<anonymous>)
    at step (C:\Users\andbo\.atom\packages\linter-pylint\lib\main.js:7)
    at processTicksAndRejections (internal/process/task_queues.js:86)
```
I am using a new atom 1.47.0 installation on windows 10 and have installed `python 3.6.0` via `pyenv 1.2.4` and `pylint 2.5.2` via `pip 20.1.1`. I have not altered any pylint settings. They're all on default. Does anyone have an idea what the problem could be?

**Comment by andb0t** [comment]
Anyone has a hint for me?

**Comment by Kwasniok** [comment]
Hey @andb0t,
have you checked your configuration? It seems as if there was a `-` inserted in the executable path.

<img width="575" alt="atom-pylint-config2" src="https://user-images.githubusercontent.com/7516208/134975625-4ce1d73c-88a9-4ab0-8e69-0e230a0f65c7.png">

# Option 1
To make it work you need to add the python scripts folder to your system environment PATH variable as well.

I installed python 3.9 via the Microsoft store. And it was located at
`C:\Users\<USER>\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\Scripts`

*NOTE: The exact path might vary on your machine!*

You might use `Get-Command python` in the PowerShell to get some hint about its location. Sadly, the `Scripts` folder might be located somewhere else then right next to the `python.exe`.

# Option 2
Inserting
`C:\Users\<USER>\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\Scripts\pylint.exe`
in the executable path field works as well.

<img width="603" alt="atom-pylint-config" src="https://user-images.githubusercontent.com/7516208/134976007-cc3ec07e-02a9-41f9-a337-79537f4b991f.png">

The same works for [python-black](https://atom.io/packages/python-black) as well if you are interested.

# Option 3 (does not work yet on Win10 but does on other systems)
Sadly inserting simply `python -m pylint` does not work (yet) on Win10. This would be my personal preference.

The latter does not seem to be the fault of this package as the same issue occurs for python-black as well.

Since, your post is a bit older I hope you have fixed it already. At least this could help someone else.
